deploy/cloud/README.md

```diff
@@ -4,29 +4,40 @@
 Provide an easy and deterministic way to setup latest cloud environment so it can be monitored and used properly.
 
 **Prerequisite**
-* [terraform](https://www.terraform.io/)
+* [Terraform](https://developer.hashicorp.com/terraform/downloads)
+* the AWS CLI, [installed](https://docs.aws.amazon.com/cli/latest/userguide/getting-started-install.html) and [configured](https://docs.aws.amazon.com/cli/latest/userguide/cli-chap-configure.html)
+* [AWS IAM Authenticator](https://docs.aws.amazon.com/eks/latest/userguide/install-aws-iam-authenticator.html)
+* the [Kubernetes CLI](https://kubernetes.io/docs/tasks/tools/install-kubectl/), also known as `kubectl`
+
```

Comment:
we can add these prerequisites to our hermit tool, and it will also pin the version so we won't have issues with different versions